man/fi_mr.3.md

```diff
@@ -637,15 +642,37 @@ is not available (because, for example, the calling process has not
 mapped the memory region into its address space), base_addr can be
 set to NULL.
 
-The selection of dmabuf over the mr_iov field is controlled by
-specifying the FI_MR_DMABUF flag.
+The selection of dmabuf over the mr_iov field and the base_mr field is
+controlled by specifying the FI_MR_DMABUF flag. The FI_MR_DMABUF flag
+cannot be used in conjunction with the FI_MR_SUBMR flag.
+
+## base_mr
+
+Sub-MR (a.k.a. memory window) registration is used to create a new MR
+from an existing MR.  The base_mr field is used to specify the MR from
+which the sub-MR is created.  The sub-MR inherits the memory backing
+the base MR and thus allows light weight memory registration becasue
+memory pinning can be avoided.  The sub-MR can limit the access rights
+and the range within the base MR.
```

Comment:
I believe that differs from memory windows, where rights can be added.  That enables added security.  Otherwise, security is based on whether the peer can guess the key to the larger region.

Reply:
Makes sense. Will update accordingly.

Reply:
Btw, the lifetime should be described.  The larger region is still accessible by a peer ever after it's destroyed as long as there's any derived region.  That's not intuitive.  IMO, it would be better to fail destruction while there are any derived regions opened to make it clear that it does still exist.